entities/library.rb

```diff
@@ -0,0 +1,34 @@
+class Library
+  include Storage
+
+  require './autoload'
+
+  attr_reader :authors, :books, :readers, :orders
+
+  def initialize
+    @authors = YAML.load_file(("./data/authors.yml"), permitted_classes: [Author])
```

Comment:
move YAML.load_file.... to some multipurpose method in module Storage and use here